pkg/controller/istio/helper.go

```diff
@@ -3,27 +3,58 @@ package istio
 import (
 	"crypto/sha256"
 	"encoding/hex"
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"k8s.io/apimachinery/pkg/runtime/schema"
+	"fmt"
 	"os"
 	"strconv"
 	"strings"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
 )
 
 var (
+	// VirtualServiceGVK => definition of virtual service GVK for oneagent
 	VirtualServiceGVK = schema.GroupVersionKind{
 		Group:   "networking.istio.io",
 		Version: "v1alpha3",
 		Kind:    "VirtualService",
 	}
 
+	// ServiceEntryGVK => definition of virtual service GVK for oneagent
 	ServiceEntryGVK = schema.GroupVersionKind{
 		Group:   "networking.istio.io",
 		Version: "v1alpha3",
 		Kind:    "ServiceEntry",
 	}
 )
 
+// CheckIstioService - Checks if Istio is installed
+func CheckIstioService(cfg *rest.Config) error {
+
+	// Creates the dynamic interface.
+	dynamicClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return err
+	}
+
+	namespace := os.Getenv(k8sutil.WatchNamespaceEnvVar)
+	//  List all of the Virtual Services.
+	virtualServices, err := dynamicClient.Resource(schema.GroupVersionResource{
```

Comment:
We should place the resource also as static definition in `var` block of line `#18`.